linked_list/linked_list.py

```diff
@@ -23,13 +19,38 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    newNode = Node(value)
+    if self.head == None:
+      self.head = newNode
+      self.tail = newNode
 
-  def remove_head(self):
-    pass
+    elif self.head != None:
+      self.tail.next_node = newNode
+      self.tail = self.tail.next_node
 
+  def remove_head(self):
+    if self.head != None:
+      return_value = self.head.value
+      self.head = self.head.next_node
+      return return_value
+    return None
+    
   def contains(self, value):
-    pass
+    currentNode = self.head
+    while currentNode != None:
+      if currentNode.value == value:
+        return True
+      currentNode = currentNode.next_node
+    return False
 
   def get_max(self):
-    pass
+    if self.head == None:
+      return None
+    maxNode = self.head.value
+    currNode = self.head
+    while currNode is not None:
+      if currNode.value > maxNode:
+        maxNode = currNode.value
+      currNode = currNode.next_node
+    return maxNode
+
```

Comment:
Nice clean code here Anne..think about commenting some of this code...but that's a nitpick